content/post/190801-mockups.md

```diff
@@ -0,0 +1,57 @@
+---
+title: "Creating mockups for SONAR"
+date: 2018-08-01T17:41:58+02:00
+draft: false
+tags: ["mockups", "user interface", "user-centered design"]
+---
+
+The work on designing the user interface of SONAR is progressing. According to the adopted methodology (presented in the beginning of the [post about personas](/post/personas/)), we are proceeding in five layers: 1. Strategy, 2. Scope, 3. Structure, 4. Skeleton and 5. Surface.
+
+Mockups belongs to layer 4, consisting in defining the *skeleton* of the system, or how it visually interacts with a user.
+
+<!--more-->
+
+### Mockup: definition
+
+In user centered design and user interface design, mockups are visual simulations of the targeted software product. They are created in order to acquire feedback during the design process. There are some terminology debates about the difference between wireframes, mockups and prototypes[^1]: the main distinction criteria are the visual fidelity and the possibility to interact with the simulation.
+
+In the project, we use the term "mockups": we would categorize them somewhere between low and high fidelity, with no interaction possible.
+
+### The wider context of user-centered design
+
+Prior to create the mockups, a whole series of scenarios for SONAR have been identified and documented in detail (layer 2: *Scope*). Each scenario is attributed to one particular persona. Examples:
+
+* Persona *Gordana* (researcher): upload a full-text publication, search publications using various criteria
+* Persona *Gabriele* (librarian): create an account for a researcher, validate an upload and add administrative data
+* Etc.
+
+Then we organised these scenarios logically, creating a tree for information architecture and a basic navigation map (layer 3: *Structure*).
+
+For the layer 4 *Skeleton*, we started to create 3-4 mockups, each simulating one scenario.
+
+### Example of one SONAR mockup
+
+The mockup presented below addresses the workflow of uploading a full-text publication as a researcher (IR as a service feature).
+
+<img class="image fit" src="/images/upload_publication.gif" alt="Mockups of uploading a full-text publication" title="Mockups of uploading a full-text publication" >
```

Comment:
FYI, there's a HUGO shortcode for pictures: https://gohugo.io/content-management/shortcodes/#figure
It helps a lot (I think).

Reply:
As it is a GIF and we had layout problems last time, I just copied the code of the other page.